File: bench/01_fibonacci/bench.py
import time

def fib(n):
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)

t0 = time.time()
r = fib(38)
ms = (time.time() - t0) * 1000
print(f"result: {r}  time: {ms:.0f}ms")

File: bench/01_fibonacci/bench.rs
use std::time::Instant;

fn fib(n: u64) -> u64 {
    if n <= 1 { n } else { fib(n - 1) + fib(n - 2) }
}

fn main() {
    let t0 = Instant::now();
    let r = fib(38);
    let ms = t0.elapsed().as_millis();
    println!("result: {}  time: {}ms", r, ms);
}
